ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -532,6 +532,26 @@ def test_ipa_dns_systemrecords_check(self):
             assert check["result"] == "SUCCESS"
             assert check["kw"]["key"] in SRV_RECORDS
 
+    def test_ipa_healthcheck_without_trust_setup(self):
+        """
+        This testcase checks that when trust isn't setup
+        between IPA server and Windows AD, IPADomainCheck
+        displays key value as domain-check and result is SUCCESS
+        """
+        msg = '0 trusts matched'
+        result = self.master.run_command(
+            ["ipa", "trust-find"], raiseonerr=False
+        )
+        assert result.returncode == 1
+        assert msg in result.stdout_text
+        returncode, data = run_healthcheck(
```

Comment:
You can probably remove the trust-find bit since this check runs whether trust is enabled or not.

This check ensures that the provide in sssd.conf for the IPA domain is "ipa".

Reply:
@rcritten removed the trust-find command from the code.